<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewWindow</class>
 <widget class="QMainWindow" name="PreviewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Captured screenshot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0">
    <property name="spacing">
     <number>16</number>
    </property>
    <property name="leftMargin">
     <number>16</number>
    </property>
    <property name="topMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>16</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editButton">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save as...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uploadButton">
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="uploadProgress">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="settingsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="settingsAction">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="icon">
    <iconset theme="qt"/>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
